front/lib/development.ts

```diff
@@ -1,13 +1,45 @@
-import type { WorkspaceType } from "@dust-tt/types";
-import { isDevelopment } from "@dust-tt/types";
+import type { LightWorkspaceType, UserType } from "@dust-tt/types";
+import { Err, isDevelopment, Ok } from "@dust-tt/types";
 
 export const PRODUCTION_DUST_WORKSPACE_ID = "0ec9852c2f";
 const PRODUCTION_DUST_APPS_WORKSPACE_ID = "78bda07b39";
 
-export function isDevelopmentOrDustWorkspace(owner: WorkspaceType) {
+export function isDevelopmentOrDustWorkspace(owner: LightWorkspaceType) {
   return (
     isDevelopment() ||
     owner.sId === PRODUCTION_DUST_WORKSPACE_ID ||
     owner.sId === PRODUCTION_DUST_APPS_WORKSPACE_ID
   );
 }
+
+export async function forceUserRole(
+  user: UserType,
+  owner: LightWorkspaceType,
+  role: "user" | "builder" | "admin"
+) {
+  // Ideally we should check if the user is dust super user but we don't have this information in the front-end
+  if (!isDevelopment()) {
```

Comment:
Let's drop this :+1:

Reply:
If i remove this, I will show the menu to everyone.
I would like to show it only for dustSuperUser but it's not an information that I have in the front end.

Reply:
(i'll merge as it but I'm happy to improve later)

Reply:
Yeah true. Let's hard-code our workspace Id here. This is fine :)

Reply:
We really really really want that in production for us

Reply:
If we don't want to expose isDustSuperUser in the UserType, what about an endpoint that returns true / false ?

Reply:
Otherwise:
(isDevelopement() || workspaceID == DUST_WORKSPACE_ID) but maybe we'd like to use it on other workspaces in prod (I imagine some of us have personal ones ?)

Reply:
All work. The easiest one that gets us 80% of the value is simply to hard code the workspace id when not in dev 👍

We should not use production personal workspaces for testing anyway :-)